Formula/elasticsearch@7.6.rb

```diff
@@ -0,0 +1,150 @@
+class ElasticsearchAT76 < Formula
+  desc "Distributed search & analytics engine"
+  homepage "https://www.elastic.co/products/elasticsearch"
+  url "https://github.com/elastic/elasticsearch/archive/v7.6.2.tar.gz"
+  sha256 "6ff4871dcae6954e13680aefc196da574a59a36418d06a7e095550ce81a370f8"
+
+  head do
+    url "https://github.com/elasticsearch/elasticsearch.git"
+  end
+
+  depends_on "openjdk"
+
+  def cluster_name
+    "elasticsearch_#{ENV["USER"]}"
+  end
+
+  def install
+    # Build the package from source
+    system "./gradlew", ":distribution:archives:oss-no-jdk-darwin-tar:assemble"
+
+    mkdir "tar" do
+      # Extract the package to the tar directory
+      system "tar", "--strip-components=1", "-xf",
+        Dir["../distribution/archives/oss-no-jdk-darwin-tar/build/distributions/elasticsearch-oss-*.tar.gz"].first
+
+      # Install into package directory
+      libexec.install "bin", "config", "lib", "modules"
```

Comment:
```suggestion
      libexec.install "bin", "lib", "modules"
      (etc/"elasticsearch").install Dir["config/*"]
```

and updating the config `inreplace`'s below